features/networking/ovn_winc.feature

```diff
@@ -0,0 +1,56 @@
+Feature: OVNKubernetes Windows Container related networking scenarios
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-26360
+  @admin
+  Scenario: Ensure Pods and Service communication across window and linux nodes
+    Given the env has hybridOverlayConfig enabled
+    And the env is using windows container
+    Given I have a project
+    And I obtain test data file "networking/list_for_pods.json"
+    When I run the :create client command with:
+      | f | list_for_pods.json |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | name=test-pods |
+    And evaluation of `pod` is stored in the :linux_pod clipboard
+    
+    Given I use the "test-service" service
+    And evaluation of `service.ip(user: user)` is stored in the :linux_service_ip clipboard
+    # Run level needs to set to 1 for windows pod to be spawned in a test project
+    When I run the :label admin command with:
+      | resource | namespace                |
+      | name     | <%= project.name %>      |
+      | key_val  | openshift.io/run-level=1 |
+    Then the step should succeed
+    Given I obtain test data file "networking/windows_pod_and_service.yaml"
+    When I run the :create client command with:
+      | f | windows_pod_and_service.yaml |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | app=win-webserver |
+    And evaluation of `pod` is stored in the :windows_pod clipboard
+    
+    Given I use the "win-service" service
+    And evaluation of `service.ip(user: user)` is stored in the :windows_service_ip clipboard
```

Comment:
could we use `service.url` which should include the port and also quote the address for IPv6

Reply:
hadn't thought about IPv6 dual-stack in the context of Windows Containers, is it supported?

Reply:
Windows nodes have no plan to support IPv6 for now at least till 4.8 or up. I would keep this as is as we might need different ways to handle ipv6 later specifically for windows node which is still not clear to us.

Reply:
I guess it is more a question of `curl` syntax on Windows, which should be governed by RFC, but as it should be the same `curl` code base I would expect `curl` to act the same on Windows and Linux.

Looks like it is defined by RFC
https://docs.microsoft.com/en-us/windows/win32/wininet/ip-version-6-support

Just tested Windows 10 with official `curl-7.71.1-win64-mingw` and  `curl.exe http://[fe80::1:1%18]/` works

Reply:
Ah I understand your confusion now. Yea so both linux and win pod honor same syntax. Would have been nicely represented if ipv6 was involved. Its just Ipv4 here with the pod.ip or service.ip clipbaord, not the host url so guess it should be fine and by default i believe,curl <ip_address> adds http as prefix if none defined.